hack/boilerplate/boilerplate.go.txt

```diff
@@ -12,4 +12,4 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
-*/
\ No newline at end of file
+*/
```

Comment:
It looks like /boilerplate.go.txt (the boilerplate used by kubebuilder) is still using `//` based comments, it should probably be updated to match the style used here as well.

Reply:
I should be able to remove that file and use the standard `boilerplate.go.txt` in its place. That would be the right thing to do instead of fixing the file.

Reply:
I removed that file altogether. now the repo has one set of boilerplate files in `hack/boilerplate` These are templates that get applied to autogenerated files and used to verify headers on others.